### ISSUE TITLE
Enhancement: Auto close opened box on Accept/Reject/Fuzzy

### ISSUE THREAD

**Issue by maximejobin** [Mte90/GlotDict#234]
If I display the "Waiting" list, I see that the list is closed like this:

![image](https://user-images.githubusercontent.com/1080760/58660742-2374d080-82f4-11e9-915d-c33c4b300506.png)

... which makes it easy to approve (or not) faster. But when I click on a button (Accept/Reject/Fuzzy), it will automatically open the next translation like this:

![image](https://user-images.githubusercontent.com/1080760/58660843-65057b80-82f4-11e9-9a35-7a5095cb9655.png)

It would be nice to prevent this behaviour so that my "approval flow" is not disrupted.

**Comment by Mte90** [comment]
DO you mean that clicking the button on the right there is the switch?
I will check that!

**Comment by maximejobin** [comment]
Clicking the button on the right applies the status. Then, I think that GlotPress itself opens the next translation "box". I would classify my issue as a bug.

What would be nice is that you close the box that auto opens.

So when I approve a string, for instance, I notice this happened in the HTML:
![image](https://user-images.githubusercontent.com/1080760/58713084-de9e7780-838f-11e9-9c34-c44cbd1c2d23.png)

In other words, you simply have to call:

    $gp.editor.hide();

... and this will do exactly what I'm hoping! :-)

**Comment by maximejobin** [comment]
@Mte90 Any explanation on why it was closed?

**Comment by garrett-eclipse** [comment]
@maximejobin it looks like @Mte90 addressed your issue in commit [#1c85248](https://github.com/Mte90/GlotDict/commit/1c85248744bb1c5bcda817ba8df5856eaf167e75) which is now live in version 1.5.7 - https://github.com/Mte90/GlotDict/releases/tag/1.5.7

When your extension updates you can test to confirm.

**Comment by Mte90** [comment]
Yep sorry for not commenting 😂

**Comment by maximejobin** [comment]
You are fast!! Wow.

I'm hitting refresh like crazy until I get the update!!

**Comment by garrett-eclipse** [comment]
@maximejobin you'll probably need to restart your browser to get the update or go into extensions to manually update
